source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,66 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take an
+additional options object as the second parameter. Set the ``fullDocument``
+field of the additional options object to ``'updateLookup'`` to receive
+an event that contains the changes to the document as well as the entire
+altered document. Alternatively, you can set ``fullDocument`` to the
+string ``'default'`` to only receive the altered document, but not the
+changes to the document. 
+
+Process the Change Stream Events
+--------------------------------
+
+You can capture events from a change stream instance by using a listener
+function. Call the watch() command to get a Change Stream instance. Add
+your listener function by calling the `EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_
+method on that instance. Set the value of the first parameter to the
+string ``'change'`` and add your :mdn:`callback function
+<Glossary/Callback_function>` as the second parameter.  The callback
+triggers when a change event is emitted, providing the next available
+document. You can specify logic in the callback to process the event
+document when it is received. 
+
+Call the :node-api:`close() </ChangeStream.html#close>` method on the
+change stream instance to stop processing change events. This method
+closes the change stream and frees resources.
+
+Example
+-------
+
+The following example opens a ``Change Stream`` on the ``movies``
+collection. We create a ``changeListener`` variable that is listener
```

Comment:
If you omit the `consts` that never get used (per earlier comment), you could make this more clear by mentioning that you're using `changeStream.on()` to define a listener function which will print change events when they occur. Youv'e already mentioned `EventEmitter.on()` so this could serve as a reminder to the reader.